### PR TITLE
Add leap exercise

### PR DESCRIPTION
**Pull request by BNAndras**
Related to #236

### CODE REVIEW

File: exercises/practice/leap/leap.coffee [hunk 1]
```diff
@@ -0,0 +1,4 @@
+class Leap
+  leapYear: (year) ->
```
Comment (ErikSchierboom) [inline exercises/practice/leap/leap.coffee:2]: Could this be a class method?

```suggestion
  @leapYear: (year) ->
```

**Review by ErikSchierboom — APPROVED** [review]

File: exercises/practice/leap/leap.coffee
Comment (ladokp) [inline exercises/practice/leap/leap.coffee:1]: It looks like the tests are not synchronized with this change.

Comment (BNAndras) [reply exercises/practice/leap/leap.coffee:1]: Can you clarify please? It seems all the expected tests are present.
```
./bin/configlet sync --tests -e leap  
Updating cached 'problem-specifications' data...
Checking exercises...
The `leap` exercise has up-to-date tests!
```

Comment (ladokp) [reply exercises/practice/leap/leap.coffee:1]: The initial code:
`class Leap
  @leapYear: (year) ->

module.exports = Leap`

produces the following error:
`TypeError: leap.leapYear is not a function`

Remove the @ from the solution and this error is gone.

Comment (BNAndras) [reply exercises/practice/leap/leap.coffee:1]: Alright, I can submit a PR for Erik’s review to fix the stubs.

Comment (ladokp) [reply exercises/practice/leap/leap.coffee:1]: The fixes are already merged for your latest exercises. 😀

**Pull request merged**